src/dfx-core/src/util/mod.rs

```diff
@@ -9,3 +15,31 @@ pub fn expiry_duration() -> Duration {
     // 4 minutes accounts for possible replica drift
     Duration::from_secs(60 * 4)
 }
+
+pub struct ByteSchema;
+
+impl JsonSchema for ByteSchema {
+    fn schema_name() -> String {
+        "Byte".to_string()
+    }
+    fn schema_id() -> Cow<'static, str> {
+        Cow::Borrowed("byte_unit::Byte")
+    }
+    fn json_schema(_: &mut SchemaGenerator) -> Schema {
+        Schema::Object(SchemaObject {
+            instance_type: Some(vec![InstanceType::Integer, InstanceType::String].into()),
+            number: None,
+            string: Some(Box::new(StringValidation {
+                pattern: Some("^[0-9]+( *([KkMmGgTtPpEeZzYy]i?)?B)?$".to_string()),
```

Comment:
The e2e test shows that `8b` is allowed. But this regex denies it. The final `B` can be lowercase.